nav2_regulated_pure_pursuit_controller/src/regulated_pure_pursuit_controller.cpp

```diff
@@ -164,6 +165,228 @@ double calculateCurvature(geometry_msgs::msg::Point lookahead_point)
   }
 }
 
+void RegulatedPurePursuitController::computeDynamicWindow(
```

Comment:
I think this should have a struct return type that is the dynamic window limits. This implicit return thing is a bit awkward